json_schemas/services-g6-saas.json

```diff
@@ -89,12 +86,10 @@
       "type": ["number", "null"]
     },
     "priceUnit": {
-      "type": "string",
-      "pattern": "^(Unit|Person|Licence|User|Device|Instance|Server|Virtual machine|Transaction|Megabyte|Gigabyte|Terabyte)$"
+      "enum": ["Unit", "Person", "Licence", "User", "Device", "Instance", "Server", "Virtual machine", "Transaction", "Megabyte", "Gigabyte", "Terabyte"]
     },
     "priceInterval": {
-      "type": "string",
-      "pattern": "^(||Second|Minute|Hour|Day|Week|Month|Quarter|6 months|Year)$"
+      "enum": ["", "Second", "Minute", "Hour", "Day", "Week", "Month", "Quarter", "6 months", "Year"]
```

Comment:
Yes it does, I've got a test that I can show you but I'm know sure it's worth committing.